.protolint.yaml

```diff
@@ -44,6 +44,10 @@ lint:
       - ENUM_FIELDS_HAVE_COMMENT
       # This one doesn't like single word RPCs, which I have no problem with
       - RPC_NAMES_CASE
+      # In practice this creates a lot of meaningless comments, we can trust
+      # authors to determine when comments will be useful or not
+      - MESSAGES_HAVE_COMMENT
```

Comment:
Can't argue with the fact that the comments this lint rule has encouraged are basically useless.